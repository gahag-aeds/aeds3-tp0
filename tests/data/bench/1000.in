1000 500
49 59 3 12 45 51 12 40 84 88 33 52 94 0 38 1 88 48 52 99 2 56 77 34 88 14 16 61 74 91 27 75 2 30 40 99 33 4 92 18 92 25 22 39 77 61 92 17 9 44 16 63 1 45 97 89 59 65 3 86 56 30 61 10 13 53 10 46 58 2 16 2 79 39 41 56 52 33 25 61 78 42 24 31 87 73 20 99 38 75 85 47 58 98 9 71 52 71 17 62 73 34 64 4 25 6 12 77 39 38 90 69 80 14 0 19 39 73 70 29 48 55 28 6 54 38 29 6 9 47 20 35 33 84 39 10 42 52 87 34 42 77 3 74 43 4 45 34 29 16 63 77 71 92 36 25 30 65 83 39 64 3 26 97 40 66 59 82 70 98 16 64 27 72 38 70 28 83 56 57 99 20 34 71 12 22 48 94 88 84 85 52 39 12 2 79 30 61 14 52 12 30 16 39 2 6 62 82 89 18 39 89 38 26 12 2 48 12 96 88 48 34 41 88 98 95 19 28 8 85 32 20 16 0 60 70 6 22 53 95 92 44 36 31 70 0 85 19 65 34 7 13 68 0 53 66 47 73 46 56 58 30 28 26 30 88 49 88 62 54 35 55 98 72 38 21 24 23 40 89 9 99 3 29 52 8 95 99 81 93 55 92 75 84 18 5 24 19 45 39 25 81 46 24 53 84 97 29 59 37 71 21 88 74 50 40 82 98 40 16 91 47 8 19 31 78 76 8 50 22 47 75 3 45 51 8 29 48 37 40 37 8 61 78 34 12 18 69 10 10 37 53 58 97 72 41 27 1 49 77 23 48 5 26 93 8 86 74 9 23 67 46 84 28 24 70 92 95 91 54 5 28 60 15 25 84 57 5 85 58 34 8 59 91 86 4 0 72 79 9 48 46 7 84 26 84 54 71 31 46 77 36 26 37 52 4 74 61 9 59 19 43 20 78 35 6 83 87 31 14 48 31 12 55 15 38 91 69 61 22 67 39 11 94 76 63 50 50 76 59 62 95 54 82 26 41 40 61 28 23 75 76 54 87 84 69 77 75 91 91 98 10 30 9 56 58 24 6 61 0 65 23 47 72 57 73 13 49 34 94 25 61 70 31 0 54 53 78 82 44 69 80 6 51 41 63 9 17 69 22 69 87 97 16 59 6 42 24 8 76 18 33 90 89 64 90 95 17 20 77 13 41 9 20 92 50 83 54 19 4 28 88 91 26 57 2 32 99 27 40 27 45 25 17 34 90 8 82 59 80 59 25 22 69 45 14 71 80 20 43 84 49 31 28 75 88 30 59 87 57 52 67 3 77 84 89 19 44 71 79 25 83 56 47 52 1 13 75 33 86 70 69 35 2 97 62 42 28 73 82 85 25 49 40 3 85 30 74 30 53 5 7 36 61 6 40 14 71 68 47 57 38 17 44 40 14 58 35 94 32 17 80 57 18 20 12 3 2 87 85 56 92 44 92 54 50 85 68 74 53 68 31 91 85 28 32 51 86 19 46 18 88 78 28 6 98 40 61 53 79 99 9 72 95 53 78 98 38 98 72 91 66 55 83 51 83 67 55 70 38 1 40 26 79 20 84 29 13 97 82 92 48 43 16 44 49 94 42 39 93 66 83 59 21 18 63 5 37 70 27 75 71 67 53 2 88 37 31 53 86 66 45 35 61 62 31 62 8 25 2 53 91 85 65 12 55 80 69 92 50 96 67 21 16 20 75 56 9 58 9 47 24 6 34 38 20 65 0 29 90 54 82 33 91 47 98 46 79 67 38 29 16 57 50 32 29 77 40 90 36 1 38 12 7 72 50 28 90 3 9 32 9 91 66 53 91 64 99 70 83 90 52 99 99 54 83 81 32 75 71 68 76 61 80 36 86 83 64 76 38 73 60 47 16 26 0 59 42 52 82 26 94 34 77 93 88 13 74 20 88 98 88 17 11 21 53 97 4 69 25 42 42 86 41 10 64 42 22 7 46 4 85 40 38 14 33 78 27 60 99 68 10 39 37 21 12 90 71 68 59 96 10 53 34 52 15 99 46 37 58 92 41 43 84 31 9 17 62 89 29 61 57 39 52 94 13 17 36 84 85 47 80 96 0 15 0 67 66 98 5 24 42 98 19 26 30 28 95 44 17 25 5 26 16 9 72 29 78 8 13 64 55 46 12 7 13 64 75 79 62 32 55 56 30 74 82 12 54 29 56 24 54 13 50 23 75 75 4 53 83 18 17 91 16 81 50 29 45 77 60 
Sub 548 880
Min 230 562
Sub 198 530
Sum 426 758
Add 404 736
Min 289 621
Sub 375 707
Min 303 635
Sub 16 348
Max 425 757
Sub 634 966
Sum 577 909
Sub 375 707
Sum 355 687
Add 548 880
Min 531 863
Sub 159 491
Max 433 765
Sub 536 868
Min 478 810
Add 294 626
Min 33 365
Add 51 383
Max 103 435
Sub 196 528
Min 174 506
Add 531 863
Max 619 951
Sub 162 494
Sum 623 955
Add 651 983
Sum 267 599
Sub 410 742
Min 181 513
Add 249 581
Sum 146 478
Add 26 358
Min 624 956
Sub 5 337
Min 41 373
Sub 605 937
Sum 195 527
Sub 618 950
Min 312 644
Sub 550 882
Max 498 830
Sub 400 732
Max 265 597
Add 487 819
Sum 268 600
Sub 75 407
Min 337 669
Sub 524 856
Max 126 458
Add 440 772
Max 276 608
Add 299 631
Max 303 635
Add 227 559
Max 17 349
Sub 482 814
Max 210 542
Add 283 615
Sum 224 556
Sub 390 722
Max 89 421
Sub 593 925
Sum 415 747
Sub 45 377
Max 155 487
Add 12 344
Max 206 538
Sub 349 681
Min 629 961
Add 74 406
Min 455 787
Add 16 348
Max 473 805
Sub 53 385
Sum 502 834
Sub 297 629
Min 127 459
Sub 396 728
Sum 377 709
Add 252 584
Min 168 500
Add 652 984
Min 107 439
Sub 27 359
Min 435 767
Add 124 456
Min 508 840
Add 201 533
Min 630 962
Sub 289 621
Max 147 479
Sub 51 383
Min 214 546
Add 561 893
Min 319 651
Sub 592 924
Sum 130 462
Add 524 856
Min 274 606
Add 91 423
Max 598 930
Sub 285 617
Sum 107 439
Add 627 959
Min 337 669
Sub 165 497
Min 481 813
Sub 464 796
Min 560 892
Add 190 522
Min 272 604
Sub 284 616
Max 399 731
Add 379 711
Max 534 866
Sub 166 498
Min 502 834
Sub 375 707
Max 367 699
Add 21 353
Max 220 552
Sub 513 845
Min 72 404
Add 657 989
Max 631 963
Add 614 946
Sum 582 914
Sub 93 425
Min 471 803
Sub 412 744
Min 407 739
Add 401 733
Max 29 361
Add 513 845
Sum 584 916
Sub 366 698
Min 235 567
Add 397 729
Max 386 718
Add 127 459
Min 144 476
Sub 657 989
Min 635 967
Add 542 874
Sum 368 700
Add 501 833
Min 264 596
Sub 426 758
Max 432 764
Sub 187 519
Sum 447 779
Sub 651 983
Sum 33 365
Add 75 407
Sum 641 973
Sub 220 552
Max 21 353
Add 112 444
Sum 536 868
Sub 491 823
Sum 523 855
Sub 275 607
Min 448 780
Sub 551 883
Min 345 677
Add 589 921
Min 473 805
Add 579 911
Min 318 650
Add 191 523
Max 262 594
Sub 461 793
Min 181 513
Add 229 561
Sum 559 891
Sub 105 437
Min 63 395
Sub 156 488
Max 531 863
Add 51 383
Sum 342 674
Sub 534 866
Sum 35 367
Sub 598 930
Sum 165 497
Add 232 564
Min 150 482
Sub 185 517
Max 343 675
Add 242 574
Min 233 565
Sub 198 530
Max 74 406
Add 237 569
Max 305 637
Add 418 750
Sum 158 490
Sub 111 443
Sum 553 885
Sub 476 808
Max 525 857
Add 147 479
Max 493 825
Sub 38 370
Min 600 932
Add 385 717
Min 367 699
Add 219 551
Max 540 872
Add 454 786
Min 269 601
Sub 438 770
Min 320 652
Add 527 859
Max 369 701
Add 484 816
Sum 91 423
Sub 75 407
Min 225 557
Add 433 765
Min 317 649
Add 457 789
Max 187 519
Add 390 722
Sum 27 359
Add 492 824
Min 364 696
Sub 408 740
Min 159 491
Add 473 805
Min 440 772
Add 16 348
Min 619 951
Sub 266 598
Max 278 610
Sub 108 440
Max 207 539
Sub 644 976
Sum 326 658
Add 303 635
Min 430 762
Add 152 484
Max 73 405
Add 306 638
Max 11 343
Add 441 773
Max 346 678
Sub 102 434
Sum 516 848
Sub 570 902
Sum 91 423
Add 389 721
Min 584 916
Sub 278 610
Sum 335 667
Sub 610 942
Sum 326 658
Add 31 363
Sum 574 906
Add 484 816
Min 60 392
Sub 347 679
Min 499 831
Add 11 343
Min 169 501
Add 333 665
Min 343 675
Sub 375 707
Min 309 641
Sub 1 333
Max 10 342
Add 295 627
Max 376 708
Add 82 414
Sum 600 932
Add 387 719
Min 80 412
Add 502 834
Sum 45 377
Sub 595 927
Min 405 737
Add 135 467
Sum 316 648
Add 321 653
Sum 412 744
Sub 207 539
Min 487 819
Add 70 402
Sum 461 793
Sub 393 725
Max 110 442
Sub 520 852
Min 296 628
Sub 609 941
Min 87 419
Add 464 796
Max 274 606
Sub 429 761
Sum 395 727
Sub 140 472
Max 654 986
Add 628 960
Sum 658 990
Add 362 694
Max 43 375
Add 148 480
Sum 81 413
Add 346 678
Max 649 981
Sub 125 457
Max 273 605
Add 386 718
Max 484 816
Add 145 477
Sum 263 595
Sub 228 560
Max 575 907
Sub 306 638
Min 522 854
Add 368 700
Min 32 364
Add 631 963
Sum 220 552
Sub 551 883
Max 129 461
Add 306 638
Min 572 904
Sub 627 959
Min 68 400
Sub 299 631
Max 99 431
Sub 54 386
Max 260 592
Sub 313 645
Max 639 971
Sub 526 858
Max 460 792
Add 277 609
Sum 204 536
Add 442 774
Max 625 957
Add 33 365
Sum 384 716
Sub 441 773
Min 401 733
Sub 204 536
Min 2 334
Sub 308 640
Min 509 841
Sub 80 412
Max 8 340
Sub 97 429
Sum 43 375
Add 451 783
Min 648 980
Sub 172 504
Min 89 421
Sub 552 884
Sum 131 463
Add 380 712
Min 407 739
Add 453 785
Min 657 989
Add 666 998
Sum 98 430
Sub 629 961
Sum 81 413
Sub 115 447
Max 238 570
Add 284 616
Sum 556 888
Add 195 527
Min 540 872
Sub 496 828
Min 193 525
Add 497 829
Min 256 588
Sub 418 750
Sum 137 469
Sub 98 430
Sum 497 829
Add 368 700
Sum 194 526
Sub 539 871
Min 375 707
Sub 398 730
Min 267 599
Sub 464 796
Min 282 614
Add 621 953
Sum 176 508
Sub 22 354
Max 406 738
Sub 166 498
Min 244 576
Add 61 393
Min 351 683
Sub 325 657
Max 175 507
Sub 145 477
Max 608 940
Add 287 619
Sum 563 895
Add 38 370
Max 339 671
Sub 88 420
Max 534 866
Add 595 927
Min 505 837
Add 228 560
Min 25 357
Add 334 666
Max 152 484
Sub 442 774
Sum 430 762
Sub 527 859
Max 173 505
Add 253 585
Min 323 655
Sub 454 786
Max 207 539
Sub 55 387
Min 32 364
Sub 169 501
Max 444 776
Add 276 608
Sum 229 561
Add 127 459
Max 600 932
Add 627 959
Max 495 827
Add 97 429
Min 470 802
Sub 391 723
Max 358 690
Add 142 474
Min 478 810
Sub 225 557
Max 69 401
Sub 316 648
Sum 452 784
Sub 317 649
Max 206 538
Sub 603 935
Min 315 647
Sub 192 524
Min 19 351
Sub 487 819
Sum 445 777
Add 341 673
Max 498 830
Sub 651 983
Sum 620 952
Sub 86 418
Min 152 484
Add 396 728
Min 361 693
Add 605 937
Min 80 412
Add 517 849
Sum 564 896
Add 131 463
Max 171 503
Sub 367 699
Min 521 853
Sub 528 860
Sum 508 840
Add 332 664
Min 309 641
Sub 133 465
Min 204 536
Sub 93 425
Max 6 338
Add 633 965
Max 253 585
Add 320 652
Max 158 490
Sub 271 603
Max 364 696
Add 602 934
Min 169 501
Sub 194 526
Max 142 474
Add 345 677
Max 279 611
Sub 293 625
Max 297 629
Add 197 529
Min 454 786
Sub 376 708
Min 270 602
Add 351 683
Max 507 839
Sub 440 772
Min 50 382
