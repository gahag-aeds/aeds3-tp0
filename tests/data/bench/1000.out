0
16812
-1
-2
98
16741
15903
-1
98
-3
-3
98
-4
98
16649
15467
-5
15175
-1
-4
15029
-4
99
98
15514
-5
96
99
100
98
98
15795
99
15815
98
98
-8
-8
99
15525
-4
14914
-4
-4
-9
-9
-9
100
-4
-8
15254
-5
97
15103
-7
-7
-8
-7
95
98
-7
95
99
-5
100
16069
-9
-10
100
16304
-9
96
-5
-7
15012
-8
97
15258
14961
16398
100
16126
15825
-12
-12
-12
-12
99
-12
15955
-2
100
13757
15477
14775
-11
94
-11
100
98
15174
15445
98
98
-10
-10
98
-8
-9
98
15174
-9
-8
99
15051
-6
-7
-6
-2
102
98
14977
-6
99
101
105
17167
14848
-3
15855
15715
16418
-3
-3
-4
-3
-4
102
106
16498
-2
15653
-2
16388
17527
-1
17445
101
-3
-1
108
16650
98
16086
102
16226
99
109
110
17631
99
-6
-1
16769
100
-6
-1
100
109
99
107
16352
99
17440
-3
-2
-3
103
15022
-8
-5
14864
-2
-9
14501
14889
110
16567
-9
-5
-3
15333
17133
15413
-3
-4
-4
15060
108
-6
-4
97
98
15889
108
108
-3
-8
99
17244
98
-3
106
-9
108
15799
97
109
-5
108
-6
100
16290
103
-6
-8
15616
105
14758
-9
-6
-9
15608
97
-8
15815
-3
-9
102
106
97
106
-10
95
107
106
-5
-4
106
-10
